src/routes.cjs

```diff
@@ -30,8 +30,13 @@ module.exports = async(fastify, _) => {
     optionalTrailingSlash('get', '/product/:slug', {schema: {
         params: {slug: {type: 'string'}}
     }}, async(request, response) => response.type('application/javascript').send(parcelTemplate.replace(
-        'const slug = null;', `const slug = '${request.params.slug}';`
-    )));
+        'const slug = null;', `const slug = '${request.params.slug}';`).replace(
```

Comment:
1. Break line just before `).replace(`, so that it appears on its own line and correct indentation
2. Look at the code
3. Inhale deeply and enjoy the satisfaction

Reply:
:)